### DISCUSSION TITLE
CCXT Pro

### DISCUSSION THREAD

**Discussion by JeromeWilson6** [community/community#33106] [Other Feature Feedback, Questions, & Ideas]
hello,I saw that "CCXT Pro will become a free part of CCXT in Q3 2022" in the website of CCXT Pro, does it has been free now? How can I install it, I try to install it through 'https://github.com/kroitor/ccxt.pro.git', however ,it does not work.

Thank you very!

**Comment by jfullstackdev** [comment]
if this is the same repo we are referring to, then you can ask them at the Issues tab

https://github.com/ccxt/ccxt/issues
